resources/sts/4.16/hypershift/openshift_hcp_capa_controller_manager_credentials_policy.json

```diff
@@ -0,0 +1,240 @@
+{
+    "Version": "2012-10-17",
```

Comment:
It looks like this file has an extra indentation level that is different from `resources/sts/4.15/hypershift/openshift_hcp_capa_controller_manager_credentials_policy.json`

Reply:
Thanks Taylor. Updated it.